RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -4,5 +4,4 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-let game = Game()
-game.start()
+MukJjiPpa(from: RockPaperScissors().start()).start()
```

Comment:
전에도 말씀드렸지만 무조건 짧은코드가 좋은 코드라고 할 수는 없습니다.
MukJjiPpa 가 RockPaperScissors().start() 를 파라미터로 받는데 해당 줄만 보면 어떤걸 말하고 싶은건지 한눈에 잘 안읽히는 것 같네요

```suggestion
let RockPaperScissorsResult = RockPaperScissors().start()
MukJjiPpa(from: RockPaperScissorsResult).start()
```

이런 식으로 한번 풀어서 작성하면 가독성이 좀더 좋아질 것 같네요 😃

Reply:
오 정말 가독성이 훨씬 좋네요!!! 감사합니다 릴라!!!😃